### PR TITLE
add twitter feed updates section

### PR DESCRIPTION
**Pull request by shreyaagrawal0809**

### CODE REVIEW

File: samantar_platform/templates/index.html [hunk 1]
```diff
@@ -16,13 +16,25 @@
   <title>Samantar</title>
 
   <!-- Font Awesome Icons -->
-  
+  <style type="text/css">
```
Comment (Deepthi-Chand) [inline samantar_platform/templates/index.html]: Please add this css files

File: samantar_platform/templates/index.html [hunk 2]
```diff
@@ -16,13 +16,25 @@
...
+    .twitter-civic{
```
Comment (Deepthi-Chand) [inline samantar_platform/templates/index.html]: Please add this css files

File: samantar_platform/templates/index.html [hunk 3]
```diff
@@ -16,13 +16,25 @@
...
 </head>
 
 <body id="page-top">
 
- 
+ <div>
+
+<div class="twitter-civic">
+  
+  <a class="twitter-timeline" data-width="220" data-height="400" href="https://twitter.com/CivicDatalab?ref_src=twsrc%5Etfw">Tweets by CivicDatalab</a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
```
Comment (Deepthi-Chand) [inline samantar_platform/templates/index.html]: Add twitter stream from cdl_samantar twitter handle

**Review by Deepthi-Chand — CHANGES_REQUESTED** [review]

**Comment by akki2825** [comment]
solve #14 #15

**Pull request merged**